### ISSUE TITLE
curlcommand.get() throws exception if request has no data

### ISSUE THREAD

**Issue by charleswhchan** [Teebytes/TnT-Fuzzer#13]
https://github.com/Teebytes/TnT-Fuzzer/blob/master/tntfuzzer/curlcommand.py#L8
  
The following line throws an exception if `self.data` is `None`     
```
        return "curl -X" + self.method.upper() + " -H \"Content-type: application/json\" -d '" + self.data + "' " \
               + self.url
```

**Comment by xenobyte** [comment]
confirmed. Thank you @charleswhchan

**Comment by xenobyte** [comment]
Solved with pull request #14

**Comment by charleswhchan** [comment]
Thanks for the quick fix.
